view-samples/shopping-cart/README.md

```diff
@@ -0,0 +1,56 @@
+# Shopping Cart
+
+## Summary
+
+The JSON template is designed for an online shopping cart. It displays items within the cart, including product details, pricing, shipping information, and user interactions like adding or removing items.
+
+Clicking on 'Add to Cart' updates the 'AddedToCart' field to true and changes the display text to 'Remove from Cart'
+
+![screenshot of the sample](./assets/screenshot.png)
+
+## View requirements
+
+Column Name               | Type
+--------------------------|----------------------------------------
+Title                     | Single line of text
+Description               | Multiple lines of text
+Price                     | Currency
+OldPrice                  | Currency
+ImageUrl                  | Single line of text
+Offer                     | Single line of text
+CalculatedPrice           | Calculated (calculation based on other columns)(Formula = OldPrice-Price)
+Rating                    | Number
+DollarValue               | Calculated (calculation based on other columns)(Formula = INT(Price))
+CentsValue                | Calculated (calculation based on other columns)(Formula = MOD(Price*100,100))
```

Comment:
I set the `Price` to 19.99 and the `CentsValue` was 98.9999999999998. Can this be reproduced in your environment? Also, would it be possible to fix it...?

![image](https://github.com/user-attachments/assets/c6fb6c53-161c-44f2-9320-4d86f692f81a)

Reply:
I have updated the formula and the type. Also use the updated JSON